### ISSUE TITLE
0.19: $hash part is missing !

### ISSUE THREAD

**Issue by dbegit** [ranisalt/node-argon2#132]
### Steps to reproduce
```
var argon2 = require('argon2');
var password = 'password';
var salt = Buffer.alloc(16, 'salt');

argon2.hash(password, {salt}).then(hash => {
  console.log("hash=" + hash);
}).catch(err => {
  console.log("err=" + err);
});
```

### Expected behaviour
It would return
$argon2i$v=19$m=4096,t=3,p=1$**c2FsdHNhbHRzYWx0c2FsdA**$_Iv3dSMJ431p24TEj68Kxokm/ilAC9HfwREDIVPM/1/0_
src: https://github.com/ranisalt/node-argon2/blob/master/test/test.js

### Actual behaviour
It only returns 
$argon2i$v=19$m=4096,t=3,p=1$**c2FsdHNhbHRzYWx0c2FsdA**
$algorithm_id$param=value(,p=v...)$**salt**$_hash_
The last part $_hash_ is missing ($_Iv3dSMJ431p24TEj68Kxokm/ilAC9HfwREDIVPM/1/0_) !
no error displayed (no error to catch).

It's the same behaviour with argon2d and argonid:
argon2i {type: argon2.argoni, salt} : 
hash=$argon2i$v=19$m=4096,t=3,p=1$**c2FsdHNhbHRzYWx0c2FsdA**

argon2d {type: argon2.argond, salt} : hash=$argon2d$v=19$m=4096,t=3,p=1$**c2FsdHNhbHRzYWx0c2FsdA**

argon2id {type: argon2.argonid, salt} :
hash=$argon2id$v=19$m=4096,t=3,p=1$**c2FsdHNhbHRzYWx0c2FsdA**

### Environment
**Operating system**:
windows 7 64bits

**Node version:**
8.7.0

**Compiler version:**
windows-build-tools@2.3.0
Microsoft Visual C++ 2015 (14.0.24210)
Microsoft Visual C++ Build Tools (14.0.25420.1)

**Comment by dbegit** [comment]
```
  var argon2 = require('argon2themax');
  var password = 'password';
  var salt = Buffer.alloc(16, 'salt');
  var options={"hashLength":32,"timeCost":3,"memoryCost":12,"parallelism":1,"type":1,"raw":false};
  argon2.hash(password, salt, options).then(function(hash) {
     console.log(hash);
  });
```

It works with argon2themax module, I have the same hash
$argon2i$v=19$m=4096,t=3,p=1$**c2FsdHNhbHRzYWx0c2FsdA**$_Iv3dSMJ431p24TEj68Kxokm/ilAC9HfwREDIVPM/1/0_

**Comment by ranisalt** [comment]
Can you try 0.19.1? Windows compilers have been a pain recently...

**Comment by dbegit** [comment]
Thanks for your help.

With 0.19.1, in all these cases:
argon2.hash(password)
argon2.hash(password, {salt})
argon2.hash(password, {type: argon2.argoni, salt})
argon2.hash(password, {type: argon2.argond, salt})
argon2.hash(password, {type: argon2.argonid, salt})
the error catched is "TypeError: opts must be an object",

**Comment by TRAHOMOTO** [comment]
@ranisalt how can I help you to fix fresh release lib?

**Comment by ranisalt** [comment]
This issue has been fixed but currently #134 is happening.
